pages/accountLists/[accountListId]/reports/donations.page.tsx

```diff
@@ -3,12 +3,16 @@ import Head from 'next/head';
 import { useTranslation } from 'react-i18next';
 import { Box } from '@material-ui/core';
 import PageHeading from '../../../../src/components/PageHeading';
-import { MonthlyActivitySection } from '../../../../src/components/Reports/DonationsReport/MonthlyActivitySection';
-import { DonationsReportTable } from '../../../../src/components/Reports/DonationsReport/DonationsReportTable';
+import { DonationsReport } from '../../../../src/components/Reports/DonationsReport/DonationsReport';
+
 import Loading from '../../../../src/components/Loading';
 import { useAccountListId } from '../../../../src/hooks/useAccountListId';
+import { GetDashboardQuery } from '../../GetDashboard.generated';
 
-const DonationsReportPage = (): ReactElement => {
+interface Props {
+  data: GetDashboardQuery;
+}
+const DonationsReportPage = ({ data }: Props): ReactElement => {
```

Comment:
I think this is a better way to type components. You get the return value and the `children` prop for free.
```suggestion
const DonationsReportPage: React.FC<Props> = ({ data }) => {
```